### DISCUSSION TITLE
Fabric Server Issue (mod issue)

### DISCUSSION THREAD

**Discussion by Paxx3** [FabricMC/fabric-api#2607] [Player support] [answered]
What's up? Any help is appreciated!

I got a problem with opening a modded server to play with my friend. I created a modpack from curseforge and it works well. I seriously don't know ALL of my mods are working and i don't know how to check the fastest method for wich mod is incompatible. Minecraft opens on launcher and i can play when i am trying to play singleplayer. I know it's a mod issue when i try to open a server but i also wanted to ask if there is a way to check faster. I don't want to use hamachi or radmin vpn because they are laggy and stuff. So i tried to open a server and got this issue. CMD just stops and freezes, sometimes crash, doing nothing else. I tried to solve the problem and after 5 minutes my head froze like CMD too. i tried my best tho :) Like i said any help is appreciated.

Here is the CMD problem in text. (pastebin) After (...8 More) CMD crashes. But i'm faster :)

https://pastebin.com/cq9XGyxs

Here is my mod pack. You can see all the mods. (259) I got a lot of mods so, i'm so sorry :/

![Modpack 1](https://user-images.githubusercontent.com/116649398/197786430-2ea94d0f-0404-488b-b65d-dcbdb1f56621.png)
![Modpack 2](https://user-images.githubusercontent.com/116649398/197786435-d03646de-4dbe-4be5-81ba-75d31956ab57.png)

In the pastebin, you can see a sentence (Sistem Belirtilen Dosyayı bulamıyor) (My language) So that means (The System Cannot Find the File Specified)

I can record the issue or anything. Just ask.

**Comment by Technici4n** [comment, marked as answer]
It's basically the same issue as https://github.com/TeamTwilight/twilightforest-fabric/issues/123. I would suggest removing the TF for now.

**Reply by Paxx3** [reply]
Thank you so much for this man. Have a great day!
